pages/services/edge-lb/1.4/tutorials/customizing-templates/index.md

```diff
@@ -0,0 +1,144 @@
+
+    For example:
+
+    ```text
+    acl itadmin-auth http_auth_group(basic-auth) admin-users
+    http-request auth realm itadmin unless itadmin-auth
+    ```
+
+1. Create additional access control list (ACL) rules inside `frontend` section to grant limited access to the users who do not belong to the `admin-users` or `regular-users` groups that you defined in the `userlist`.
+
+    For example:
+
+    ```text
+    acl guest-users hdr_dom(host) -i guest.example.org
+    acl basic-auth-user http_auth(basic-auth)
+    acl basic-auth-user-with-group  http_auth_group(basic-auth) admin-users regular-users
+    http-request auth realm guest if guest-users !basic-auth-user OR guest-users basic-auth-user-with-group
+    use_backend web-guest-production if guest-users
+    ```
+
+1. Verify your user list and the access control rules you have set for the `frontend` and `backend` sections.
+
+For example:
+
+    ```text
+    userlist basic-auth
+      group regular-users
+      group admin-users
+
+      user admin  password ASYRtiLFCipT6 groups admin-users
+      user drew password n5se4LwUfAW1sqA groups regular-users
+      user luz password x18R29iAdV/$1QU groups regular-users
+      user guest insecure-password guestpassword
+
+    frontend web
+      bind :80
+      #bind :443 ssl crt /etc/ssl/cert/
+
+      option httplog
+      log /dev/log local0 debug
+
+      option forwardfor except 127.0.0.1
+      option forwardfor header X-Real-IP
+
+      #redirect scheme https code 301 if !{ ssl_fc }
+
+      # ACL rule for general access for users defined in backend
+      acl example hdr_dom(host) -i example.org
+      use_backend web-example-production if example
+
+      # ACL rule for admin-users group member defined in the backend
+      acl itadmin hdr_dom(host) -i itadmin.example.org
+      use_backend web-example-production if itadmin
+
+      # ACL rules for guests who are not admin-users or regular-users
+      acl guest-users hdr_dom(host) -i guest.example.org
+      acl basic-auth-user http_auth(basic-auth)
+      acl basic-auth-user-with-group  http_auth_group(basic-auth) admin-users regular-users
+      http-request auth realm guest if guest-users !basic-auth-user OR guest-users basic-auth-user-with-group
+      use_backend web-guest-production if guest-users
+
+    backend web-example-production
+      acl example-auth http_auth(basic-auth)
+      http-request auth realm example unless example-auth
+
+      mode http
+      server example 10.0.10.15:80
+
+    backend web-itadmin-production
+      acl itadmin-auth http_auth_group(basic-auth) admin-users
+      http-request auth realm itadmin unless itadmin-auth
+
+      mode http
+      server itadmin 10.0.10.16:80
+
+    backend web-guest-production
+      mode http
+      server guest 10.0.10.17:80
+    ```
+
+1. Create an Edge-LB pool to use the custom template by running a command similar to the following:
```

Comment:
This is incorrect - we already created an edge-lb pool, we just need to update its template. The correct command for this is:
```
dcos edgelb template update <name of the pool that we created> haproxy.tmpl
```

Reply:
Changed